pkg/version/version.go

```diff
@@ -0,0 +1,19 @@
+package version
+
+import "fmt"
+
+// ComboVersion indicates what version of Combo the binary belongs to
+var ComboVersion string
```

Comment:
I think I'm more-or-less interested in the git tag + commit information for this version sub-command - wdyt? Maybe rename this variable to reflect the git tag here and update the root Makefile to set this ldflag explicitly if we're fine with that layout.

Reply:
I realize there's no guarantee we're processing an unannotated tag, especially for HEAD commits, so maybe we need to use `git describe ...` here to grab lightweight tags.

Reply:
Using Git tags here sounds like a much better solution than the static version file. I would like to see a little automation around bumping that version or at least making sure that we are bumping it manually with a tag each time. Have you done anything like that before? Probably out of the scope of this PR.

Reply:
Yeah, automation would be nice, but it's not something we have to iron out immediately. I'm like to get a structure in place before introducing automation when the manual process could be fairly trivial. Longer term, we'll likely want to introduce something like goreleaser to handle all of this for us anyways.

Reply:
Okay, I've replaced the static file with a `git describe` in the Makefile. It also has a default version since we currently do not have any tags.